rootfs/api/models/appsettings.py

```diff
@@ -19,7 +19,9 @@ class AppSettings(UuidAuditedModel):
     app = models.ForeignKey('App', on_delete=models.CASCADE)
     maintenance = models.NullBooleanField(default=None)
     routable = models.NullBooleanField(default=None)
-    whitelist = ArrayField(models.CharField(max_length=50), default=[])
+    # the default values is None to differentiate from user sending an empty whitelist
+    # and user just updating other fields meaning the values needs to be copied from prev release
+    whitelist = ArrayField(models.CharField(max_length=50), default=None)
```

Comment:
Won't this break the client since it expects an array?

https://github.com/deis/workflow-cli/blob/master/cmd/whitelist.go#L24-L26
https://github.com/deis/controller-sdk-go/blob/master/api/appsettings.go#L25